leads/lead_monitoring.md

```diff
@@ -63,6 +63,52 @@ These are just a few of the integrations already created by the Mautic community
 
 **Note:** It is important to note that you are not limited by these plugins and you can place the tracking pixel directly on any HTML page for website tracking.
 
+### Mobile Monitoring
+
+The essence of monitoring what happens in an App is similar to monitoring what happens on a website. Mautic contains the building blocks needed for native (or pseudo-native) and HTML5-wrapper based Apps, regardless of platform.
+
+In short, use named screen views (e.g. main_screen) in your App as your page_url field in the tracker, and the lead's email as the unique identifier, see next section for detailed instructions.
+
+
+#### Steps in Mautic
+
+First, setup a form, which will be the access point of your campaign (e.g. a new lead email). Make this form as simple as you can, as you will be POST-ing to it from your App. The typical form URL you will POST to is 
+
+```
+http://your_mautic/form/submit?formId=<form_id>
+```
+
+You can get the ID from the mautic URL as you view / edit the form in the Mautic interface (or in the database tables, last column), and you can get the form fields by looking at the HTML of the 'Manual Copy' of the HTML in the forms editing page.
```

Comment:
I should have made myself more clear. I didn't mean the database table (although it would work as well, but ID in that table is the first column). I meant this table:
![mautic-form-ids](https://cloud.githubusercontent.com/assets/1235442/10578354/e479bae0-7671-11e5-9b48-869cd2bd59ca.png)